modules/integration/tests-common/admin-clients/pom.xml

```diff
@@ -272,6 +268,13 @@
             <groupId>org.wso2.carbon.registry</groupId>
             <artifactId>org.wso2.carbon.registry.properties.stub</artifactId>
         </dependency>
+        <!-- Using the previous stub version for compilation due to deprecation of the challenge question management feature in IS 7.0.0 and onwards -->
+        <dependency>
+            <groupId>org.wso2.carbon.identity.governance</groupId>
+            <artifactId>org.wso2.carbon.identity.recovery.stub</artifactId>
+            <version>1.8.87</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
this would be temporary yes?

Reply:
yes it's. After hosting the challenge questions as a connector we need to improve the test cases to download it and run the test on top of it. Afterwards we can remove this.

Reply:
shall we add a comment here explaining that?

Reply:
fixed with https://github.com/wso2/product-is/pull/17772/commits/9d22f845ed9a6e826e9ee34f70d395b7e1a67edf